modules/social_features/social_event/src/Entity/Node/Event.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace Drupal\social_event\Entity\Node;
+
+use Drupal\social_event\EventEnrollmentInterface;
+use Drupal\social_node\Entity\Node as Node;
+
+/**
+ * Defines bundle class for event node type.
+ */
+final class Event extends Node implements EventInterface {
```

Comment:
So by choosing this to be a "final" class, are we right that we don't want any one to extend this?

Reply:
Yes, correct. Make class as final tells developers that `Event` behaviour should be only in `Event` itself - if you want to have something the same as `Event` just make double-think if you really need to have it. It save us  from having classes with similar behaviour which "extending" allows.

Reply:
@nechai One of the philosophy we want to follow in Open Social that we want our open source users to be able to extend the these classes. If they are extending, they have to be well aware of the development practices and what could go wrong. 

I do understand your approach and why you wanted to mark it as final, but we can leave upon at the discretion of community users/developers whether they want to extend it or not.

Reply:
@navneet0693 removed `final` modifier 🙂